Paintroid/src/main/java/org/catrobat/paintroid/FileIO.java

```diff
@@ -51,9 +51,9 @@
 
 public final class FileIO {
 	public static String filename = "image";
-	public static String ending = ".jpg";
+	public static String ending = ".png";
 	public static int compressQuality = 100;
```

Comment:
it uses the compressQuality for:
`bitmap.compress(compressFormat, compressQuality, outputStream)`
This line also gets called for PNG compression. I am not sure if the compress function, just ignores the quality parameter if the format is PNG. Since PNG uses lossless compression.